geoparser_app/static/js/app.js

```diff
@@ -600,6 +614,54 @@ $(function() {
 			})
 
 })
+
+$(function() {
+    $("#viewIndexResults").keypress(function(e) {
+        if(e.which == 13) {
+            $("#searchResults").empty();
+            var keyword = $(this).val();
+            var searchLink = "http://localhost:8984/solr/scientific_content_enrichment/select?indent=on&q="+keyword+"&wt=json";
```

Comment:
Looks like your local change. May be you can do it in a generic way by showing dropdowns of available indexes as we do currently with view index button.